C#/Sample Projects/Navigation/DriveAndDock/README.txt

```diff
@@ -0,0 +1,31 @@
+The basic outline of the system is:

```

Comment:
We might change the extension for readme files to `.md` instead of .txt; .txt files display fine, but Markdown plays nicely and looks a bit pettier when you are viewing a repository online.

Reply:
Done